tools/lsp/main.rs

```diff
@@ -102,6 +105,22 @@ struct Format {
     inline: bool,
 }
 
+#[cfg(feature = "preview-engine")]
+#[derive(Args, Clone, Debug)]
+struct LivePreview {
+    /// Run remote controlled by the LSP
+    #[arg(long)]
+    remote_controlled: bool,
+    /// Run remote controlled by the LSP
+    #[arg(long, default_value = "")]
+    style: String,
```

Comment:
Wrong doc comment

Reply:
I kept the fullscreen one, the rest comes via configuration requests anyway.